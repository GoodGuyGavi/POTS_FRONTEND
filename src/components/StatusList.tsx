import React from "react";

import styled from "styled-components";
import StatusItem from "./StatusItem";
import HeaderTitle from "../components/HeaderTitle";

export default class StatusList extends React.Component {
  render() {
    return (
      <div style={{ marginLeft: 20, marginTop: 10 }}>
        <HeaderTitle header="Today's Status" />
        <StatusItem
          OR="3"
          label="Orders Received"
          label2="Released from supplier"
          label3="On-hold from supplier"
        />
      </div>
    );
  }
}
