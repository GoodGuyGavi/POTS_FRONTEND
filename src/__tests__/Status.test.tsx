/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
import StatusReportList from "../components/Status";
import StatusList from "../components/StatusList";

const props = {
  title: "Todayâ€™s Status",
  status: [
    {
      id: 1,
      number: "3",
      statOption: "Orders Received"
    },
    {
      id: 2,
      number: "7",
      statOption: "Released from supplier"
    },
    {
      id: 3,
      number: "2",
      statOption: "On-hold from supplier"
    }
  ]
};

const statusList = {
  title: "Title",
  onPress: () => console.log("Pre")
};

describe("Status Report List", () => {
  test("renders correctly", () => {
    expect(shallow(<StatusReportList {...props} />)).toMatchSnapshot();
  });
});

describe("Status", () => {
  test("renders correctly", () => {
    expect(shallow(<StatusList {...statusList} />)).toMatchSnapshot();
  });
});
