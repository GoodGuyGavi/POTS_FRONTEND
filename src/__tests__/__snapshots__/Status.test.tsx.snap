// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status Report List renders correctly 1`] = `ShallowWrapper {}`;
